Solutions/Infoblox Cloud Data Connector/Analytic Rules/Infoblox-HighNumberOfHighThreatLevelQueriesDetected.yaml

```diff
@@ -0,0 +1,42 @@
+id: 57113ad7-7dd6-4150-84d8-252e162aaf4a
+name: Infoblox - High Number of High Threat Level Queries Detected
+description: |
+  'This creates an incident in the event a single host generates at least 200 high threat level RPZ queries (Threat Defense security hits) in 1 hour. Query count threshold and scheduling is customizable. This rule depends on a parser based on a Kusto Function to work as expected called [**InfobloxCDC**](https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/Infoblox%20Cloud%20Data%20Connector/Parsers/InfobloxCDC.txt).'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: InfobloxCloudDataConnector
+    dataTypes: 
+      - CommonSecurityLog (InfobloxCDC)
```

Comment:
Since you are using the parser in queries, you will have to add its schema in json format under [CustomTable folder.](https://github.com/Azure/Azure-Sentinel/tree/master/.script/tests/KqlvalidationsTests/CustomTables) Only than the validations would pass.

Reply:
@NikTripathi I see, do I have to include every field including the fields MS parses for me, or just the fields that I parse manually in the InfobloxCDC parser?

Reply:
@sschuur Every field.

Reply:
@NikTripathi Thank you!! added